### PR TITLE
Update function get_sitemap_urls on magazine /models.py

### LINKED ISSUE #644: (PYL-W0221) Mismatched parameters in overridden method
## Description
The parameters in the overridden method don&#x27;t match the method defined in the parent class. There are three possible scenarios for this: - The number of parameters are different in both methods - One or more names of the parameters are different in both methods - The order of parameters …

## Occurrences
There are 2 occurrences of this issue in the repository.

See all occurrences on DeepSource &rarr; [app.deepsource.com/gh/WesternFriend/WF-website/issue/PYL-W0221/occurrences/](https://app.deepsource.com/gh/WesternFriend/WF-website/issue/PYL-W0221/occurrences/)

### PR DESCRIPTION
**Pull request by viktoriussuwandi**
fix #644

Hi [brylie](https://github.com/brylie),

on [magazine/models.py](https://github.com/WesternFriend/WF-website/blob/d34aed15eceee25321ee91b2dc7af6cce1a07857/magazine/models.py#L173),
I Add `changefreq` variable on function `get_sitemap_urls` , 
and put `None` as default value since I have no idea the `datatype` for this variable.

Please check it out

### CODE REVIEW

**Comment by brylie** [comment]
When creating a pull request that will resolve an issue, please [link the pull request to the related issue(s)](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) by using one of the [supported keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) followed by the issue number.

For example:

```
Closes #<issue-number>
```

## Further details

> You can link a pull request to an issue by using a supported keyword in the pull request's description or in a commit message.
> 
> - close
> - closes
> - closed
> - fix
> - fixes
> - fixed
> - resolve
> - resolves
> - resolved

**Comment by viktoriussuwandi** [comment]
Hi [brylie](https://github.com/brylie), just let me know if anything else needed to complete this PR

File: magazine/models.py [hunk 1]
```diff
@@ -180,9 +180,14 @@ class Meta:
             models.Index(fields=["drupal_node_id"]),
         ]
 
-    def get_sitemap_urls(self) -> list[dict]:
-        return [{"location": self.full_url, "lastmod": self.latest_revision_created_at}]
-
+    def get_sitemap_urls(self,changefreq=None,priority=None) -> list[dict]:
```
Comment (brylie) [inline magazine/models.py:183]:
```suggestion
    def get_sitemap_urls(self, changefreq=None, priority=None) -> list[dict]:
```

File: magazine/models.py [hunk 2]
```diff
@@ -344,15 +349,16 @@ class MagazineArticle(DrupalFields, Page):  # type: ignore
     parent_page_types = ["MagazineIssue"]
     subpage_types: list[str] = []
 
-    def get_sitemap_urls(self) -> list[dict]:
+    def get_sitemap_urls(self,changefreq=None,priority=None) -> list[dict]:
```
Comment (brylie) [inline magazine/models.py:352]:
```suggestion
    def get_sitemap_urls(self, changefreq=None, priority=None) -> list[dict]:
```

**Comment by viktoriussuwandi** [comment]
Thanks @brylie !

**Pull request merged**